### ISSUE TITLE
Refresh not working

### ISSUE THREAD

**Issue by hupo376787** [lucasg/Dependencies#231]
I put halconxl.dll in the same folder, and refresh it, but Dependencies still reported missing.
Restart app not working either.
![image](https://user-images.githubusercontent.com/14960168/192735287-24af4b63-3f52-494c-9e63-3e501333b4cf.png)
![image](https://user-images.githubusercontent.com/14960168/192735418-bec5654c-7dcb-4599-992c-8086ca3766d4.png)

**Comment by oold** [comment]
I just noticed that the refresh doesn't work in general. I've been using Dependencies to check the exported symbols in DLLs that I'm building and was wondering why some exported functions that I had added before refreshing were missing. Disabling the binary cache fixes that, but then I have to close the tabs before building.

TL;DR: Refreshing the binary view is broken or works in unexpected (and fairly useless) ways.
